frontend/app/component/form/differences/DifferencesForm.test.tsx

```diff
@@ -258,13 +349,36 @@ describe("Test DifferencesForm", () => {
       await user.click(submitButton);
 
       const feedbackError = await screen.findByTestId("feedback-error");
-      expect(feedbackError).toHaveTextContent(/^IncorrectTotal,IncorrectTotal$/);
+      expect(feedbackError).toHaveTextContent(/^IncorrectTotal$/);
+      expect(screen.queryByTestId("feedback-warning")).toBeNull();
+      expect(screen.queryByTestId("server-feedback-error")).toBeNull();
+    });
+
+    test("F.27 Incorrect total", async () => {
```

Comment:
Tests should use `overrideOnce()` and not rely on the mock implementation.

Reply:
Changed in b90d2a8d33a757b4a0ada6d31a3ade9b59aab4f8